src/lib/utils/formatting.ts

```diff
@@ -23,3 +23,16 @@ export const prettyBytes = (bytes: number, decimals = 2) => {
 
   return parseFloat((bytes / Math.pow(k, i)).toFixed(dm)) + ' ' + sizes[i];
 };
+
+export const prettyArch = (platform: string) => {
+  switch (platform) {
+    case 'WindowsNoEditor':
+      return 'Client';
```

Comment:
Please change to `Windows Client`

Reply:
https://github.com/satisfactorymodding/smr-frontend/pull/47/commits/9c0893e0353c97b347f9f0152ec590c614d255ac